packages/test-interface-criteria/src/criteria/criteriaUtils.js

```diff
@@ -376,8 +391,13 @@ function mustNotBeOpenBookingInAdvanceUnavailable(offer) {
 */
 function mustHaveBeInsideValidFromBeforeStartDateWindow(offer, opportunity, options) {
```

Comment:
This function name is confusing. Could now be called `mustBeInsideBookingWindowIfOneExists(..)` to mirror `mustHaveBookingWindowAndBeOutsideOfIt(..)`

Reply:
Good idea, have clarified